typescript/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+dist/
```

Comment:
might as well use this https://www.toptal.com/developers/gitignore/api/node,webstorm